api/src/main/java/com/datatorrent/api/DefaultInputPort.java

```diff
@@ -109,4 +109,13 @@ public void teardown()
    */
   public abstract void process(T tuple);
 
+  protected int getCount()
```

Comment:
Not needed, there is already `getCount(boolean reset)`

Reply:
done